### DISCUSSION TITLE
OLED care: Random noising of boundaries?

### DISCUSSION THREAD

**Discussion by soaxelbrooke** [rxhanson/Rectangle#1463] [Q&A]
I've got an OLED monitor that I use, and I am interested in minimizing OLED burn-in, which is exacerbated by consistent high contrast boundaries, AKA windows splitting the screen at exactly the same point every waking hour of the monitor's life. What I'd like to achieve is randomly fuzzing where the middle, first third, second third, etc boundaries are on some frequency to distribute burn-in on the monitor and increase the effective burn-in free life of the monitor. Is this:

a) an existing feature?
b) a thing I can create some automation around to change via config/etc?

Thanks!

**Comment by rxhanson** [comment]
There's nothing built into Rectangle to solve this problem, and it seems a bit outside the scope of functionality I would personally consider adding into Rectangle. On one hand, this seems like a great idea for a separate, focused, single purpose app, but I can also see where it would make sense for a window manager to include it since is already aware of window placement. I see no reason to turn down a good pull request with this functionality, as long as it doesn't degrade existing functionality.

**Comment by bernaferrari** [comment]
I don't think Rectangle would help much with this. You would still have the window opened taking some space. Your biggest hope is when Apple starts making Macbook with OLED next year. Will you hide the dock and the top toolbar because of oled?
